Basic/DyanmicPrefabs/README.md

```diff
@@ -0,0 +1,27 @@
+# Dynamic Prefabs
```

Comment:
Will this be updated in a followup PR?
I remember we discussed the various pros and cons for each techniques, I'd expect them listed at least here if not in code comments.
Things like "spawning with NetworkShow prevents a single client from blocking other clients, but creates different world versions for each clients. If a boss appears, you wouldn't want to use this technique as you would want to wait for everyone to load the assets before having a game changer element like a boss"
Same with other techniques.

Reply:
@jilfranco-unity has this mocked up for a follow up PR. I'll add more comments to that as well.

Reply:
Yeah, so this will go inside of this [PR](https://github.com/Unity-Technologies/com.unity.multiplayer.samples.bitesize/pull/93).

Reply:
ok, let me know when you're done there, I'll review